### PR TITLE
Source Prestashop: prevent users from using http url

### LINKED ISSUE #16520: Add check to prevent users from using http url with Prestashop
Tell us about the problem you're trying to solve
Currently, it is possible to use unsecured URL (http) in the Prestashop config.

Describe the solution you’d like
We should fail the CHECK if a user enters a non-https URL for Prestashop.

### PR DESCRIPTION
**Pull request by grubberr**
Signed-off-by: Sergey Chvalyuk <grubberr@gmail.com>

## What
*Describe what the change is solving*
*It helps to add screenshots if it affects the frontend.*

## How
*Describe the solution*

## Recommended reading order
1. `x.java`
2. `y.python`

## 🚨 User Impact 🚨
Are there any breaking changes? What is the end result perceived by the user? If yes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.

## Pre-merge Checklist
Expand the relevant checklist and delete the others.

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [x] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [x] Secrets in the connector's spec are annotated with `airbyte_secret`
- [x] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [x] Code reviews completed
- [x] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [x] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [x] PR name follows [PR naming conventions](https://docs.airbyte.com/contributing-to-airbyte/issues-and-pull-requests)

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- [x] Create a non-forked branch based on this PR and test the below items on it
- [x] Build is successful
- [x] If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).
- [x] [`/test connector=connectors/<name>` command](https://docs.airbyte.io/connector-development#updating-an-existing-connector) is passing
- [ ] New Connector version released on Dockerhub and connector version bumped by running the `/publish` command described [here](https://docs.airbyte.io/connector-development#updating-an-existing-connector)

</details>

## Tests

<details><summary><strong>Unit</strong></summary>

*Put your unit tests output here.*

</details>

<details><summary><strong>Integration</strong></summary>

*Put your integration tests output here.*

</details>

<details><summary><strong>Acceptance</strong></summary>

*Put your acceptance tests output here.*

</details>

### CODE REVIEW

**Review by davydov-d — APPROVED** [review]

File: airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/orders.json [hunk 1]
```diff
@@ -147,52 +147,50 @@
...
+              "product_ean13": {
+                "type": "string"
+              },
+              "product_isbn": {
+                "type": "string"
+              },
+              "product_upc": {
+                "type": "string"
+              },
+              "product_price": {
```
Comment (mickaelandrieu) [inline airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/orders.json:180]: I'm pretty sure this is a float, is "string" the only data type allowed at the moment?

For the record, I was part of the PrestaShop Core Team for 4 years.

Comment (grubberr) [reply airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/orders.json:180]: Hay @mickaelandrieu 
all types are supported: "string", "number",  "integer", "boolean"
are you talking about "product_price" field only ?

Comment (grubberr) [reply airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/orders.json:180]: @mickaelandrieu can you make PR  for all fields which has incorrect type ?

Comment (mickaelandrieu) [reply airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/orders.json:180]: yes I will asap, maybe during the week end :)

**Pull request merged**